problem1/himanshu.rb

```diff
@@ -1,2 +1,17 @@
 class Abc
-end
\ No newline at end of file
+	def triangle(n)
```

Comment:
@HimanshuTiwariPlum : Code indentation issue.

Reply:
@AjayBarot upate the indentation of the program.